src/indicators/flooding/glofas/glofas.py

```diff
@@ -209,7 +209,7 @@ def _get_processed_filepath(
         if leadtime is not None:
             filename += f"_lt{str(leadtime).zfill(2)}d"
         filename += ".nc"
-        return DATA_DIR / PROCESSED_DATA_DIR / country_iso3 / GLOFAS_DIR / filename
+        return DATA_DIR / PUBLIC_DATA_DIR / PROCESSED_DATA_DIR / country_iso3 / GLOFAS_DIR / filename
```

Comment:
Ah I see you've added this in here so probably don't need to make that change on the correlation branch.

Reply:
Yes I should have just made a separate PR for this, I've had to add this line in so many branches ><